templates/cases/oddsligers.html

```diff
@@ -0,0 +1,87 @@
+{% extends "base.html" %}
+{% block title %}OddSligers{% endblock %}
+{% block subtitle %}OddSligers{% endblock %}
+{% block article %}
+<div class="container-full cases-home">
+    <div class="project-detail">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12 col-lg-9">
+                    <div class="row project-header">
+                        <div class="col">
+                            <h1><a href="https://oddslingers.com">oddslingers.com</a> - Case Study (2018)</h1>
+                            <img src="/static/core/img/odd-slingers.png" style="max-height: 46px; max-width: 244px">
+                            <p>
+                                Technicality level: <b>4/5</b>
+                            </p>
+                        </div>
+
+                    </div>
+                    <div class="row">
+                        <div class="col">
+                            <img
+                                    src="/static/core/img/img-cases-002.png"
+                                    alt="Syncroplicity"
+                                    class="syncro-img"
+                            >
+                        </div>
+                    </div>
+                    <div id="about" class="row">
+                        <div class="col">
+                            <h3>About the project</h3>
+                            <p>
+                                A modern take on the aging online poker platform:
+                                new game variants, ad-free online games for desktop
+                                & mobile using React animations and django-channels
+                                websockets.<br><br>
+                                Implemented bleeding-edge technology, including an
+                                event stream system, <a class="blue-text" href="https://cirw.in/blog/time-to-move-on">MOVE</a>-like
+                                architecture, pure-functional reactive frontend animations (to do this we had to write our own
+                                <a class="blue-text" href="https://github.com/Monadical-SAS/redux-time">frontend animations library</a>),
+                                and an in-house system for time-travel debugging and introspection tools.
+                            </p>
+                        </div>
+                    </div>
+                    {% include "clients-reviews.html" %}
+                    {% include "work-with-us.html" %}
+                </div>
+{#                <div class="xs-hide col-lg-3">#}
```

Comment:
You can disable code blocks in html using <!-- commented HTML here -->

Reply:
yes I know, mi Editor comment in this way automatically.